### ISSUE TITLE
[Bug]: High Contrast: 200% magnification (Level AAA - Accessibility)

### ISSUE THREAD

**Issue by joepercival** [ONSdigital/design-system#2847]
### What happened?

This issue was raised on the Statistical Methods Library static website; [Link to High Contrast: 200% magnification ticket](https://jira.ons.gov.uk/browse/SPP-9559)

### What was the expected outcome?

See ticket

### How is this problem impacting your service?

See ticket

### What list of steps can be followed to reproduce the bug or issue?

See ticket

### What version of the ONS Design System are you using?

62.0.0

### What devices are you seeing the problem on?

- [X] Desktop
- [ ] Mobile
- [ ] Tablet

### What operating systems are you seeing the problem on?

- [ ] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Other

### What browsers are you seeing the problem on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Microsoft Internet Explorer
- [ ] Other

**Comment by adi-unni** [comment]
When the page is magnified to 200%, the links in the navigation menu there is no visible change in focus when users hover over the elements with a mouse. This issue may affect low vision users as they may be unable to distinguish that focus has been placed on them. - copying for visibility

**Comment by yatesn** [comment]
Needs some design work before solution is implemented

**Comment by JNATION85** [comment]
Need to implement the same hover effect when in mobile size
